bentoml/_internal/bento/build_config.py

```diff
@@ -472,10 +473,11 @@ def write_to_bento(self, bento_fs: "FS", build_ctx: str) -> None:
                 ]
             )
             logger.info("Locking PyPI package versions..")
-            click_ctx = pip_compile_cli.make_context("pip-compile", pip_compile_args)
+            cmd = [sys.executable, "-m", "piptools", "compile"]
+            cmd.extend(pip_compile_args)
             try:
-                pip_compile_cli.invoke(click_ctx)
-            except Exception as e:
+                subprocess.check_call(cmd)
```

Comment:
I think check_call doesn't work on windows? Maybe use check_output instead here?

Reply:
Oh, what's the issue with `check_call` on windows?

Reply:
oh nvm it was to related to this.